default/buildings.txt

```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```

Comment:
@MatGB: What Cj said. Basically don't do any dev work in master, but create a branch for each feature your working on (except for very small, quick fixes which can be done in a single commit, in that case the create branch - implement fix - merge into master workflow is a bit of an overkill).